Mathlib/Data/Matroid/Rank/ENat.lean

```diff
@@ -108,24 +114,6 @@ lemma IsBasis.eRk_eq_encard (hIX : M.IsBasis I X) : M.eRk X = I.encard := by
 lemma IsBase.eRk_eq_eRank (hB : M.IsBase B) : M.eRk B = M.eRank := by
   rw [hB.indep.eRk_eq_encard, eRank_def, hB.isBasis_ground.encard_eq_eRk]
 
-@[simp]
-lemma eRank_map {β : Type*} (M : Matroid α) (f : α → β) (hf : InjOn f M.E) :
-    (M.map f hf).eRank = M.eRank := by
-  obtain ⟨B, hB⟩ := M.exists_isBase
-  rw [← (hB.map hf).encard_eq_eRank, ← hB.encard_eq_eRank, (hf.mono hB.subset_ground).encard_image]
-
-@[simp]
-lemma eRank_loopyOn (E : Set α) : (loopyOn E).eRank = 0 := by
-  simp [← (show (loopyOn E).IsBase ∅ by simp).encard_eq_eRank]
-
-@[simp]
-lemma eRank_emptyOn (α : Type*) : (emptyOn α).eRank = 0 := by
-  simp [← (show (emptyOn α).IsBase ∅ by simp).encard_eq_eRank]
-
-@[simp]
-lemma eRk_ground (M : Matroid α) : M.eRk M.E = M.eRank :=
-  M.eRank_def.symm
-
```

Comment:
The next lemma and a few around it also reference ground and eRank; so it seems to make sense to keep them together? I don't have a strong preference though; it seems more discoverable to me to have "similar" lemmas together, but similar is a metric you can define better than I

Reply:
(I'm referring to the ones like `eRk_inter_ground` and below, it doesn't let me comment on them directly!)